### DISCUSSION TITLE
Kriging model validation process

### DISCUSSION THREAD

**Discussion by XudongF** [GeoStat-Framework/PyKrige#240] [Q&A]
Hi, I am still new to the Kriging algorithm. I wonder how to evaluate a fitted Kriging model? The cross-validation is a little confusing to me. I think I should remove a point, use the kriging model to predict that point, and repeat this process. But it seems once the model is fitted, there is no way to change the 'reference points' in PyKrige? In other words, the PyKrige model will always give the exact value of this removed point. But how to conduct the cross-validation then? Should I re-train the Kriging model by using the remaining points? 

Many thanks!

**Comment by MuellerSeb** [comment]
Hi there,

yes you should setup the Kriging model again with the left-out points.

Sebastian
